src/actions/guides/frontend/internationalization.cr

```diff
@@ -0,0 +1,526 @@
+    ## Step 2 - Add localization ymls
+
+    first make a locales folder:
+    ```
+    mkdir config/locales
+    ```
+
+    Then add at least one localization: i.e. English (here is a list of the fixed text - excluding the ones missed)
+    ```
+    # config/locales/en.yml
+    en:
+      action:
+        save_success: "The record has been saved"
+        update_success: "The record has been updated"
+        delete_success: "Deleted the record"
+        index_title: "All Records"
+        create_new: "New Record"
+        new: New
+        edit: Edit
+        delete: Delete
+        confirm: "Are you sure?"
+        update: Update
+        updating: "Updating..."
+        save: Save
+        saving: "Saving..."
+        back: Back
+        back_to_index: "Back to All"
+      auth:
+        sign_in: "Sign In"
+        sign_in_success: "You're now signed in"
+        sign_in_failure: "Sign in failed"
+        sign_up: "Sign up"
+        sign_up_success: "Thanks for signing up"
+        sign_up_failure: "Couldn't sign you up"
+        signed_out: "You have been signed out"
+        pwd_update: "Update Password"
+        pwd_update_success: "Your password has been reset"
+        pwd_reset: "Password Reset"
+        pwd_reset_request: "Reset your Password"
+        pwd_reset_req_success: "You should receive an email on how to reset your password shortly"
+      auth_token:
+        not_authenticated: "Not Authenticated."
+        invalid: "The provided authentication token was incorrect."
+        missing: "An authentication token is required. Please include a token in an 'auth_token' param or 'Authorization' header."
+      default:
+        page_name: Welcome
+      error:
+        title: "Something went wrong"
+        try_home: "Try heading back to home"
+        locked_out: "Locked-out"
+        auth_incorrect: "is wrong"
+        form_not_valid: "It looks like the form is not valid"
+        not_in_system: "is not in our system"
+      user:
+        email: E-Mail
+        profile: Profile
+        next: "Next, you may want to"
+        auth_guides: "Check out the 'Authentication Guides'"
+        modify_page: "Modify this page"
+        after_signin: "Change where you go after sign in"
+        preferred_language: "Preferred Language"
+    ```
+
+    > Add additional langauges as needed in the same folder.
+    > Be sure that all lang yml files contain the same keys. If there's no translation for that key, just leave the value blank.
+
+    ## Step 3 - Configure i18n within Lucky
+    ```
+    # config/i18n.cr
+    I18n.backend = I18n::Backends::YAML.new.tap do |backend|
+      backend.load_paths << Dir.current + "/config/locales"
+      backend.load
+    end
+    ```
+
+    ## Step 4 - Add 'lang' to users table
+
+    Generate a migration using:
+    ```
+    lucky db.migration AddLanguageToUser
+    ```
+
+    edit the file to look like (of course, the number will vary):
+    ```
+    # db/migrations/20191228100116_add_language_to_user.cr
```

Comment:
```suggestion
    # db/migrations/#{Time.utc.to_s("%Y%m%d%H%I%S")}_add_language_to_user.cr
```

I did this in a few other places https://github.com/luckyframework/website/blob/master/src/actions/guides/database/migrations.cr#L54 Not necessary, but fun 😂